moi/__init__.py

```diff
@@ -16,6 +16,24 @@
 from moi.context import Context  # noqa
 
 
+def _support_directory():
+    """Get the path of the support_files directory"""
+    from os.path import join, dirname, abspath
```

Comment:
@antgonza no, I don't think this is running on an ipengine

@wasade I really have no strong feelings, just doing my best impression of a linter :laughing: 

This PR has my :+1: